package.json

```diff
@@ -72,16 +72,15 @@
     "vrtest": "cd apps && cd vr-tests && yarn screener"
   },
   "devDependencies": {
-    "@angular-devkit/schematics": "12.0.2",
```

Comment:
not needed anymore, see https://github.com/microsoft/fluentui/pull/18748/files#r665180543

Reply:
nice catch, lemme check. AFAIR @TristanWatanabe  expanded that ignore regex because of storybook

Reply:
as expected storybook "needs it" 

![2021-08-30 at 19 55](https://user-images.githubusercontent.com/1223799/131383259-160edc16-ab89-4e33-a245-e75cec4a6a1f.png)

Reply:
That's too bad. Thanks for checking!